platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r85/ProblemsServiceModelBuilderCrossVersionTest.groovy

```diff
@@ -82,8 +83,8 @@ class ProblemsServiceModelBuilderCrossVersionTest extends ToolingApiSpecificatio
         """
     }
 
-    @TargetGradleVersion(">=8.6")
-    @ToolingApiVersion(">=8.6")
+    @TargetGradleVersion(">=8.7")
```

Comment:
❌ he test coverage should not be changed for existing releases. Instead, an upper bound should be defined (`@TargetGradleVersion(">=8.6 <8.7")`) and the new behaviour for 8.7 should be defined in a separate test class in the `r87` package.